### DISCUSSION TITLE
adding a demo/code example for drag and drop

### DISCUSSION THREAD

**Discussion by mbassiouny33** [unoplatform/uno#10438] [General]
Hi

It seems (according to a PR I found) that Drag and drop is supported in this platform. There are many examples to help people get started  in [code samples](https://platform.uno/code-samples/) and in  [the gallery](https://gallery.platform.uno/)  but IMHO they seem to lack a tutorial on Drag and drop for desktop apps, after some research I barely found it is possible but the information is buried here https://platform.uno/docs/articles/features/pointers-keyboard-and-other-user-inputs.html and without really any code to help beginners.

Would be nice to add an extra link in the gallery and/or a small app in code samples to show it

Thanks

**Comment by jeromelaban** [comment]
Thanks for the feedback! Drag and drop is based on [the docs for UWP](https://learn.microsoft.com/en-us/windows/apps/design/input/drag-and-drop), and you can find step by step details on how to implement it.
